netlify/pages/elements.yaml

```
@@ -0,0 +1,153 @@
+path: /open-source/elements
+title: Stoplight Elements
+relatedTags:
+  - blog-elements
+hero:
+  title: Open Source API Documentation Building Blocks
+  subtitle: >-
+    Build highly customized, interactive API Docs with embeddable web components generated from OpenAPI documents.
+  aligned: center
+  features:
+    - title: Open Source
+    - title: Maintained by Stoplight
+    - title: Free Forever
+  ctas:
+    - submit:
+        button:
+          title: Get Early Access
+          color: blue2
+        formId: 22bf0180-f3dd-4113-ac29-f68a7f408cb4
+        input:
+          placeholder: Your work email...
```

```suggestion
          placeholder: Your email...
```

In the footer signup we just say "Your email" so this makes it consistent.